src/Common.cpp

```diff
@@ -386,3 +387,51 @@ void SplitString(const std::string& str, const char delim, std::vector<std::stri
     }
 }
 
+std::vector<uint8_t> DeComp(std::span<const uint8_t> input) {
+    beammp_debugf("got {} bytes of input data", input.size());
+
+    std::vector<uint8_t> output_buffer(std::min<size_t>(input.size() * 5, 15 * 1024 * 1024));
```

Comment:
What about a short comment detailing the constants in the vector constructor? As someone who hasn't touched this code it isn't immediately apparent what these do.